Framework/MDAlgorithms/test/IntegratePeaksMD2Test.h

```diff
@@ -207,14 +209,18 @@ class IntegratePeaksMD2Test : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_NOTHING(alg.setProperty("BackgroundOuterRadius", bkgOuterRadius));
     TS_ASSERT_THROWS_NOTHING(alg.setProperty("Ellipsoid", ellip));
     TS_ASSERT_THROWS_NOTHING(alg.setProperty("Cylinder", cyl));
-    std::cout << "Running simpleRun with inputs: shouldPass:" << shouldPass
-              << " numberValuesPeakRadius:" << numberValuesPeakRadius
-              << " numberValuesBkgInnerRadius:" << numberValuesBkgInnerRadius
-              << " numberValuesBkgOuterRadius:" << numberValuesBkgOuterRadius << " cyl:" << cyl << " ellip:" << ellip
-              << "\n";
+    Logger logger("Logger_simpleRun");
+    logger.setLevel(Logger::Priority::PRIO_INFORMATION);
```

Comment:
One more nit, we shouldn't change the log level here as this will change the developer's selected log level by running this test